src/apple.rs

```diff
@@ -43,26 +55,46 @@ pub struct AppleResponse {
     #[serde(rename = "is-retryable")]
     pub is_retryable: Option<bool>,
     pub environment: Option<String>,
+    /// The latest Base64 encoded app receipt. Only returned for receipts that contain auto-renewable subscriptions. 
+    #[serde(rename = "latest-receipt")]
+    pub latest_receipt: Option<String>,
     /// An array that contains all in-app purchase transactions. This excludes transactions for consumable products
     /// that have been marked as finished by your app. Only returned for receipts that contain auto-renewable subscriptions.
+    #[serde(rename = "latest-receipt-info")]
     pub latest_receipt_info: Option<Vec<AppleLatestReceipt>>,
 }
 
-pub async fn validate_apple(
+/// Retrieves the responseBody data from Apple
+pub async fn apple_response(
     receipt: &UnityPurchaseReceipt,
-    client: &Client<hyper_tls::HttpsConnector<hyper::client::HttpConnector>>,
-    apple_urls: &AppleUrls,
     password: Option<&String>,
-) -> Result<PurchaseResponse> {
+) -> Result<AppleResponse> {
+    apple_response_internal(receipt, &AppleUrls::default(), password).await
```

Comment:
This could maybe be changed to apple_response_with_urls() or something, and exposed to the user as something they could use for their own tests now that I think about it